features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml

```diff
@@ -1355,6 +1355,17 @@
         <Scopes>internal_custom_authenticator_delete</Scopes>
     </Resource>
 
+    <!-- [Organization] User defined local authenticator API -->
```

Comment:
Let's move this block ahead of tenant paths since B2B API paths are also falls to tenant API path patterns